Boltdir/.rerun.json

```diff
@@ -0,0 +1 @@
+[{"target":"ip-10-227-2-25.amz-dev.puppet.net","status":"failure"}]
```

Comment:
I don't think you meant to commit this file

Reply:
:man_facepalming: I have removed it from the PR.